### PR TITLE
Azure functions .NET 6 and host v4

### PR DESCRIPTION
**Pull request by andreasohlund**

### CODE REVIEW

File: nservicebus/upgrades/azure-functions-service-bus-isolated-1to2.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    return serviceBusTriggeredEndpointConfiguration;
+});
+```
+
+use:
+
+```csharp
+functionsHostBuilder.UseNServiceBus(endpointName, nsb =>
+{
+    nsb.Routing.RouteToEndpoint(typeof(SomeMessage), "AnotherEndpoint");
```
Comment (DavidBoike) [inline nservicebus/upgrades/azure-functions-service-bus-isolated-1to2.md:62]: What is the type of `nsb`? Shouldn't it be endpointConfiguration or functionEndpointConfiguration or something?

File: nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
+title: Azure Functions (isolated) for Service Bus Upgrade Version 3 to 4
+summary: How to upgrade Azure Functions (isolated worker) with Azure Service Bus from version 3 to 5
 component: ASBFunctions
-reviewed: 2021-10-09
+reviewed: 2021-12-16
 related:
+ - nservicebus/upgrades/7to8
 isUpgradeGuide: true
 upgradeGuideCoreVersions:
  - 8
```
Comment (DavidBoike) [inline nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md:10]: Functions versions 2 and 3 are both NServiceBus Core 7, right?
```suggestion
 - 7
```

File: nservicebus/upgrades/azure-functions-service-bus-in-process-2to3.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+title: Azure Functions (in-process) for Service Bus Upgrade Version 2 to 3
+summary: How to upgrade Azure Functions (in-process) with Azure Service Bus from version 2 to 3
+component: ASBFunctions
+reviewed: 2021-12-16
+related:
+ - nservicebus/upgrades/7to8
+isUpgradeGuide: true
+upgradeGuideCoreVersions:
+ - 8
```
Comment (DavidBoike) [inline nservicebus/upgrades/azure-functions-service-bus-in-process-2to3.md:10]: Functions versions 2 and 3 are both NServiceBus Core 7, right?
```suggestion
 - 7
```

File: samples/azure-functions/service-bus-worker/ASBFunctionsWorker_4/AzureFunctions.ASBTrigger.Worker/host.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+    "version": "2.0"
```
Comment (DavidBoike) [inline samples/azure-functions/service-bus-worker/ASBFunctionsWorker_4/AzureFunctions.ASBTrigger.Worker/host.json:2]: This is the version of the host.json schema, not Azure Functions Host v2, right?

File: nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md (hunk 2 above)
Comment (andreasohlund) [reply nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md:10]: No, in-process worker v3 targets core v8 , see https://github.com/Particular/NServiceBus.AzureFunctions.InProcess.ServiceBus/blob/master/src/NServiceBus.AzureFunctions.InProcess.ServiceBus/NServiceBus.AzureFunctions.InProcess.ServiceBus.csproj#L18

File: nservicebus/upgrades/azure-functions-service-bus-in-process-2to3.md (hunk 3 above)
Comment (andreasohlund) [reply nservicebus/upgrades/azure-functions-service-bus-in-process-2to3.md:10]: In process v3 targets core v8 https://github.com/Particular/NServiceBus.AzureFunctions.InProcess.ServiceBus/blob/master/src/NServiceBus.AzureFunctions.InProcess.ServiceBus/NServiceBus.AzureFunctions.InProcess.ServiceBus.csproj#L18

File: samples/azure-functions/service-bus-worker/ASBFunctionsWorker_4/AzureFunctions.ASBTrigger.Worker/host.json (hunk 4 above)
Comment (andreasohlund) [reply samples/azure-functions/service-bus-worker/ASBFunctionsWorker_4/AzureFunctions.ASBTrigger.Worker/host.json:2]: Correct

File: nservicebus/upgrades/azure-functions-service-bus-isolated-1to2.md (hunk 1 above)
Comment (andreasohlund) [reply nservicebus/upgrades/azure-functions-service-bus-isolated-1to2.md:62]: Fixed

File: nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md (hunk 2 above)
Comment (andreasohlund) [reply nservicebus/upgrades/azure-functions-service-bus-isolated-3to4.md:10]: my bad, you are correct again

**Review by DavidBoike — APPROVED** [review]

**Comment by DavidBoike** [comment]
@andreasohlund Going to go ahead and merge this PR so that it will fix the docs errors currently occurring due to the obsoletions.

**Pull request merged**